UX/ui-service/Common/Tags/design.md

```diff
@@ -0,0 +1,45 @@
+# Tags - Common Component
+* Tags are used in multiple areas throughout the UI.
+* Tags are made up of two parts, a category and value. Some categories allow for more than one value to be associated.  
+* Tags can be applied to a single item or multiple items at one time depending on the use case.
+* In all cases, the Tag selection dropdowns and the tag labels should be visually consistent.
+* As tags are added, they should be placed at the bottom of the list. Once saved, the tags should be reordered into alphabetic order so upon returning to the tags area, users can easily scan the list.
+
+## Simple Tags
+  * ![Image of a Simple Tag.](img/SimpleTags-01.png)  
+
+1. **Selection:**
+  * A combobox should be used for the tag selection so users can rely on the dropdown or they can type to filter the list of options.
```

Comment:
@serenamarie125 I guess I don't see why we wouldn't use the BS Combobox. I didn't follow the + Search comment you made. The BS Combobox allows the user to type in order to filter the list down so I think that is what we need, right?